frontend_v2/src/app/services/global.service.ts

```diff
@@ -439,7 +440,9 @@ export class GlobalService {
     if (err.status === 401) {
       this.checkTokenValidity(err, toast);
     } else if (err.status === 403 && toast) {
-      this.showToast('error', err.error['error'], 5);
+      console.log(err.error);
```

Comment:
It is better to remove the console log before commit that may slow down the browser tab and I think we do not want to show the internal logs publicly.